index.html

```diff
@@ -49,5 +49,211 @@ <h2>I'm a Software Developer</h2>
 
     </section>
 
+    <section id="work">
+      <h3>My Recent Works</h3>
+
+      <span class="seperation"></span>
+
+      <div>
+
+        <img src="images\multi-post Stories.png">
+
+        <div class="multi-post-descript">
+
+          <h4>Multi-Post Stories</h4>
+
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+            has been the industry's standard dummy text ever ssince the 1500s,
+            when an unknown printer took a standard dummy text.
+          </p>
+
+          <ul>
+            <li>css</li>
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>ruby</li>
+          </ul>
+
+          <button type="button">See Project</button>
+
```

Comment:
- [ ] Good job :+1:  @IvanLudick , everything looks perfect however you should  add the button styling: Active, Hover and Pressed (see design) for increasing the user experience :100: , Kindly update that to that every button's in your page

### You could do for example
```bash
.button-tag::hover {
/* some properties*/
  }
```